features/car/restrictions.feature

```diff
@@ -438,7 +438,7 @@ Feature: Car - Turn restrictions
         When I route I should get
             | from | to | route                               |
             | e    | f  | ae,xa,bx,fb,fb                      |
-            | c    | f  | dc,da,ae,ge,hg,hg,ge,ae,xa,bx,fb,fb |
+            | 1    | f  | dc,da,ae,ge,hg,hg,ge,ae,xa,bx,fb,fb |
```

Comment:
```
| restriction | cb       | bx     | b        | only_right_turn  |
```
This turn restriction routes you into a one-way road going in the other direction. I'm not sure if this was intended.
In any case, I added a numbered input location to ensure it snaps to the same `dc` as previously.